packages/jsii/lib/assembler.ts

```diff
@@ -201,6 +201,7 @@ export class Assembler implements Emitter {
       readme,
       jsiiVersion,
       fingerprint: '<TBD>',
+      bin: this.projectInfo.bin,
```

Comment:
Let's try to keep `fingerprint` last here, at least for now.

```suggestion
      bin: this.projectInfo.bin,
      fingerprint: '<TBD>',
```